arcjet/index.ts

```diff
@@ -521,13 +522,24 @@ type BotOptionsDeny = {
 
 export type BotOptions = BotOptionsAllow | BotOptionsDeny;
 
-export type EmailOptions = {
+type EmailOptionsAllow = {
   mode?: ArcjetMode;
-  block?: ArcjetEmailType[];
+  allow: ArcjetEmailType[];
+  deny?: never;
+  requireTopLevelDomain?: boolean;
+  allowDomainLiteral?: boolean;
+};
+
+type EmailOptionsDeny = {
```

Comment:
```suggestion
export type EmailOptionsDeny = {
```
Need to export this to make documentation easier